libcodechecker/cmd/cmd_line_client.py

```diff
@@ -464,6 +473,67 @@ def printResult(getterFn, baseid, newid, suppr, output_format):
                     args.output_format)
 
 
+def handle_suppress(args):
+    def update_suppression_comment(run_id, report_id, comment):
+        client.unSuppressBug([run_id], report_id)
+        client.suppressBug([run_id], report_id, comment)
+
+    def bug_hash_filter(bug_id, filepath):
+        return [
+            codeCheckerDBAccess.ttypes.ReportFilter(bugHash=bug_id,
+                                                    suppressed=True,
+                                                    filepath=filepath),
+            codeCheckerDBAccess.ttypes.ReportFilter(bugHash=bug_id,
+                                                    suppressed=False,
+                                                    filepath=filepath)]
+
+    already_suppressed = 'Bug {} in file {} already suppressed. Use --force!'
+
+    client = setupClient(args.host, args.port, '/')
+
+    run_info = check_run_names(client, [args.name])
+    run_id, run_date = run_info.get(args.name)
+
+    if args.output:
+        with open(args.output, 'w') as out:
+            for suppression in client.getSuppressedBugs(run_id):
+                out.write(suppression.bug_hash + '||' +
+                          suppression.file_name + '||' +
+                          suppression.comment + '\n')
+
+    elif args.input:
+        with open(args.input, 'r') as inp:
+            for line in inp:
+                bug_id, file_name, comment = line.rstrip().split('||')
+
+                reports = client.getRunResults(run_id, 500, 0, None,
```

Comment:
The `500` value should not be hardcoded, there is a constant value in the thrift api files which should be used.